sections/main-cart-items.liquid

```diff
@@ -46,13 +46,13 @@
                 <tr class="cart-item" id="CartItem-{{ item.index | plus: 1 }}">
                   <td class="cart-item__media">
                     {% if item.image %}
+                      <a href="{{ item.url }}" class="cart-item__link" aria-hidden="true" tabindex="-1"> </a>
```

Comment:
This results in invalid HTML. Any element with an opening tag must be accompanied by a closing tag (with the exception of elements which do not feature a text node such as `input`, `img`, `br`, etc.

Reply:
You are right, thank you.  TIL - I knew with HTML5 more tags could be self-closing and assumed this was the case for `a` too.

Can we remove the extra space? 

```suggestion
                      <a href="{{ item.url }}" class="cart-item__link" aria-hidden="true" tabindex="-1"></a>
```

Reply:
Originally, I removed the extra space, but due to the [a:empty](https://github.com/Shopify/dawn/blob/f22de6feb14144c27b21b1f79162ffbfb2d87b61/assets/base.css#L515) display: none rule, it gets overwritten.

If I remove the empty space, most likely I will have to write a more specific CSS rules.
What are your thoughts?

Reply:
Oh you are right! Then I think we can leave it as is since that makes sense 👍 

And the decision/reason for it is documented here in case we need to come back to it